public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -79,13 +79,13 @@ export class ChartComponent implements OnChanges, AfterViewInit {
         outside: true,
         formatter: function(tooltip) {
             if (this.point.y <= 20) {
-               return this.point.toolTipHelpText + '<br/> <b> &le; ' + this.point.y + '</b>';
+               return this.point.toolTipHelpText + '<b> &le; ' + this.point.y + '</b>';
              }
              // If not <= 20, use the default formatter
-             return this.point.toolTipHelpText + '<br/> <b> ' + this.point.y + '</b>';
+             return this.point.toolTipHelpText;
         },
         useHTML: true,
-        backgroundColor: '#f0f2f3',
+        backgroundColor: 'transparent',
```

Comment:
![image](https://user-images.githubusercontent.com/860209/61464568-1cedf700-a93c-11e9-90c4-64b1206f865a.png)
We should have a solid background here

Reply:
Thanks for noticing this/